.github/workflows/android.yml

```diff
@@ -0,0 +1,27 @@
+name: cross compile with android-ndk
```

Comment:
I think we can just put this job into the workflow file we already have

Reply:
Ok, I will do it.